sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -1276,6 +1277,7 @@ describe("Atom management - Authentication", function(): void {
       enableBatchedOperations: false,
       status: "SendDisabled" as EntityStatus,
       enablePartitioning: true,
+      enableExpress: false,
```

Comment:
This seems fine.

One observation: 
If `requiresDuplicateDetection` and `enableExpress` are both set to true at the same time, service throws an error.
`RestError: SubCode=40000. Both DelayedPersistence property and RequiresDuplicateDetection property cannot be enabled together. To know more visit https://aka.ms/sbResourceMgrExceptions. . TrackingId:b2714073-5403-4880-813b-e7eefe5d02d0_G41, SystemTracker:harshan-servicebus:Topic:management-topic-1, Timestamp:2020-09-09T07:40:48`